apps/web/public/static/locales/en/common.json

```diff
@@ -1479,6 +1479,8 @@
   "must_enter_team_name": "Must enter a team name",
   "team_url_required": "Must enter a team URL",
   "url_taken": "This URL is already taken",
+  "domain_taken_team": "Domain is already owned by another team but you can request delegation to access it.",
+  "domain_taken_project": "Domain is already being used by a different project.",
```

Comment:
As these messages refer to Vercel projects and teams, I think it would be best to just have one single message here like `"problem_registering_domain": "There was a problem with registering the subdomain, please try again or contact an administrator"` and use it in both scenarios 🙏